files/build_templates/per_namespace/lldp.service.j2

```diff
@@ -16,9 +16,9 @@ StartLimitBurst=3
 
 [Service]
 User={{ sonicadmin_user }}
-ExecStartPre=/usr/bin/{{docker_container_name}}.sh start{% if multi_instance == 'true' %} %i{% endif %}
-ExecStart=/usr/bin/{{docker_container_name}}.sh wait{% if multi_instance == 'true' %} %i{% endif %}
-ExecStop=/usr/bin/{{docker_container_name}}.sh stop{% if multi_instance == 'true' %} %i{% endif %}
+ExecStartPre=/usr/local/bin/{{docker_container_name}}.sh start{% if multi_instance == 'true' %} %i{% endif %}
+ExecStart=/usr/local/bin/{{docker_container_name}}.sh wait{% if multi_instance == 'true' %} %i{% endif %}
+ExecStop=/usr/local/bin/{{docker_container_name}}.sh stop{% if multi_instance == 'true' %} %i{% endif %}
```

Comment:
show lldp table tries to fetch the data for all asics based on asic.conf configuration. That throws error on console for dockers not running. Is that taken care of in this or a different PR? If not, whats the expectation?

Reply:
https://github.com/Azure/sonic-utilities/pull/1656 - Have created this to skip non-existent containers. We can discuss if more changes are required as part of that PR.